public/index.html

```diff
@@ -1,36 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="utf-8" />
```

Comment:
looks like Prettier modified index.html after I removed the bootstrap CDN

Reply:
:+1: